apps/mobile-alphabet/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
Is this a development \ testing concern for being able to hit the development back-end over localhost? If so, this must be configurable. There may be differences in dev \ testing \ production in these policies. Look into this more.